riotfile.py

```diff
@@ -1328,6 +1329,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="pymysql",
             command="pytest {cmdargs} tests/contrib/pymysql",
+            env={"DD_DBM_PROPAGATION_MODE": "full"},
```

Comment:
```suggestion
```

Reply:
Flaws in our dbm procedure support were detected by running mysql tests with `DD_DBM_PROPAGATION_MODE=full`. 

I am tempted to keep this environment variable in this PR because it can serve as a regression test.

Reply:
you can parametrize the env by doing 

```
env = {"DD_DBM_PROPAGATION_MODE": ["full", ""]}
```

Reply:
Oh nice. I didn’t know that was option. I’ll use that in a follow up PR.